pkgs/top-level/aliases.nix

```diff
@@ -1224,6 +1224,7 @@ mapAliases ({
   pkgconfig = pkg-config; # Added 2018-02-02, moved to aliases.nix 2021-01-18
   pkgconfigUpstream = throw "'pkgconfigUpstream' has been renamed to/replaced by 'pkg-configUpstream'"; # Converted to throw 2022-02-22
   planner = throw "planner has been removed from nixpkgs, as it is no longer developed and still uses python2/PyGTK"; # Added 2021-02-02
+  pleaser = please; # Added 2023-03-12
```

Comment:
what do you think renaming the expression to `pleaser` and adding an alias `please = pleaser;`?

Reply:
Well, technically our package is named correctly, it's just that it's sometimes named `pleaser` to avoid collisions in some distros. I guess I can just remove all that if adding an alias wouldn't help anyway. Or maybe we can just change the `pname` and leave the actual attribute name as-is.

Reply:
These aliases are just a transitional measure.  They shouldn't be used and instead should be changed into a `throw` after some time, as you can see all around.  I don't see why introduce such an alias for a _new_ package.